homeassistant/components/viam/const.py

```diff
@@ -0,0 +1,25 @@
+"""Constants for the viam integration."""
+
+DOMAIN = "viam"
+
+CONF_API_ID = "api_id"
+CONF_SECRET = "secret"
+CONF_CREDENTIAL_TYPE = "credential_type"
+CONF_ROBOT = "robot"
+CONF_ROBOT_ID = "robot_id"
+
+CRED_TYPE_API_KEY = "api-key"
+CRED_TYPE_LOCATION_SECRET = "robot-location-secret"
+
+SERVICE_VALUES = "values"
+SERVICE_COMPONENT_NAME = "component_name"
+SERVICE_COMPONENT_TYPE = "component_type"
+SERVICE_FILEPATH = "filepath"
+SERVICE_CAMERA = "camera"
+SERVICE_CONFIDENCE = "confidence_threshold"
+SERVICE_ROBOT_ADDRESS = "robot_address"
+SERVICE_ROBOT_SECRET = "robot_secret"
+SERVICE_FILE_NAME = "file_name"
+SERVICE_CLASSIFIER_NAME = "classifier_name"
+SERVICE_COUNT = "count"
+SERVICE_DETECTOR_NAME = "detector_name"
```

Comment:
This file should only have constants which are used in more than one file. That doesn't seem to be the case for many of these, for example the `SERVICE_*` should be moved to `services.py`.